client/components/account-balances/header.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { CardHeader, Flex, FlexItem } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { getGreeting } from './utils';
+import { useCurrentWpUser } from './hooks';
+import wooPaymentsLogo from '../../../assets/images/woopayments.svg';
```

Comment:
This import target will change from `../../../assets/` to `assets/` once #5609 is merged.

Reply:
If we merge this before that PR I'm assuming we'll need to either update #5609 or come back and tidy up?

Reply:
Yeh, a bit of a race condition happening here.

This will work regardless and will have no impact on project outcomes.

I would categorise this as a minor code quality consideration only.

Reply:
I've handled it as part of the merge with `develop`, thanks for noticing it! 🙂